noxfile.py

```diff
@@ -102,7 +102,11 @@ def lint(session: Any) -> None:
     session.run("black", "--check", *SOURCE_FILES)
     session.run("flake8", *SOURCE_FILES)
 
-    lint_per_folder(session)
+    pylint_overrides = ["opensearchpy/", "test_opensearchpy/"]
```

Comment:
Instead of this find all .pylintrc files under first level subfolders and run those?